newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -154,6 +154,9 @@ const checkDirectionPointsAndCollisionsMasks = (direction: gdDirection) => {
   };
 };
 
+const removeExtensionFromFileName = fileName =>
+  fileName.substring(0, fileName.lastIndexOf('.'));

```

Comment:
Should you handle the situation where fileName does not have a dot?

If possible I think we should still fallback to the resource name?